src/components/button-toggle/button-toggle.style.ts

```diff
@@ -136,7 +139,43 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
         }
       }
       cursor: not-allowed;
-    `};
+    `}
+
+  ${({ variant, disabled }) =>
+    variant === "minor" &&
+    css`
+      & ${StyledIcon} {
+        color: var(--colorsActionMinor500);
+        :not([aria-pressed="true"]):not(:disabled):hover {
+          color: var(--colorsActionMinorYang100);
+        }
+      }
+
+      color: var(--colorsActionMinor500);
+      border: 1px solid var(--colorsActionMinor500);
+
+      :not([aria-pressed="true"]):not(:disabled):hover {
+        color: var(--colorsActionMinorYang100);
+        background-color: var(--colorsActionMinor600);
+        ${StyledIcon} {
+          color: var(--colorsActionMinorYang100);
+        }
+      }
+
+      &:not(:disabled):hover {
+        background-color: var(--colorsActionMinor600);
+        color: var(--colorsActionMinorYang100);
+
+        ${StyledIcon} {
+          color: var(--colorsActionMinorYang100);
+        }
+      }
+
+      &[aria-pressed="true"]:not(:hover) {
+        box-shadow: inset 0px 0px 0px 3px var(--colorsActionMinor500);
```

Comment:
comment: the focus styling is still not correct when a button has been pressed, you'll need to add something like this below line 176:

```

        :focus {
          box-shadow: inset 0px 0px 0px 3px var(--colorsActionMinor500), 0px 0px 0px var(--borderWidth300) var(--colorsSemanticFocus500), 0px 0px 0px var(--borderWidth600) var(--colorsUtilityYin090);
        }